odoo/addons/base/models/ir_async.py

```diff
@@ -0,0 +1,259 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import datetime
+import itertools
+import json
+import logging
+import re
+import traceback
+import threading
+import odoo
+import odoo.exceptions
+from odoo import models, fields, api
+from odoo.http import serialize_exception
+
+_logger = logging.getLogger(__name__)
+BASE_VERSION = odoo.modules.load_information_from_description_file('base')['version']
+
+
+class IrAsync(models.Model):
+    _name = 'ir.async'
+    _description = 'Asynchronous Jobs'
+
+    name = fields.Char()
+    state = fields.Selection([
+        ('created', 'Created'),     # The job has been enqueued for later processing
+        ('succeeded', 'Succeeded'), # The job finished with a result, the result has not yet been retrieved
+        ('failed', 'Failed'),       # The job failed due to an exception
+        ('done', 'Done'),           # The job finished without result or the result has been retrieved
+    ])
+    res_model = fields.Char()
+    res_ids = fields.Text()
+    method = fields.Char()
+    args = fields.Text()
+    kwargs = fields.Text()
+    user_id = fields.Many2one('res.users', string='Scheduler User', ondelete='cascade')
+    context = fields.Text()
+    super_user = fields.Boolean()
+    traceback = fields.Text()
+    payload = fields.Text(default=False)
+
+    def call(self, target, args=None, kwargs=None):
```

Comment:
Why not this?
```suggestion
    def call(self, method, *args, **kwargs):
```
The name `method` may be better than `target`, as it must be a bound method.